index.js

```diff
@@ -0,0 +1,71 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const Stocks = require('./database-mongodb/Stock.js');
+const Purchases = require('./database-mongodb/Stock2.js');
+
+const app = express();
+const PORT = 3001;
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+app.use(express.static(__dirname + '/../client/dist'));
+
+
+app.get('/api/ratings', function(req, res) {
```

Comment:
Looks like you already got the proper route set up below this one. Feel free to remove this!

Reply:
Ok thank you!